### ISSUE TITLE
`plotIndividualTimeProfile` and `plotPopulationTimeProfile` line type

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite-R#926]
Default line type should be "solid" and not "dashed".

**Comment by IndrajeetPatil** [comment]
Please don't create new issues for this. 

All these points should be mentioned in the Discussion.

**Comment by PavelBal** [comment]
I want to merge the PR ASAP and continue working on improvements in separate issues. In the discussion, there is no way to track what has been fixed.

**Comment by IndrajeetPatil** [comment]
Discussion is not the right place to see what is implemented.

It is the example plots in the PR, which are updated after each commit:

https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/871/files

![image](https://user-images.githubusercontent.com/11330453/167833860-98e55897-7fc0-4e2a-afbb-0e55548f2043.png)

**Comment by IndrajeetPatil** [comment]
Fixed in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/871/commits/0c213b77c2ce14ed79ef9f204a205306e0ae27c3

**Comment by msevestre** [comment]
It's absolutely ok to create issue for this. The PR can reference the issue

**Comment by IndrajeetPatil** [comment]
We will just have to agree to disagree then.

**Comment by msevestre** [comment]
Yep. Just don't close the issue if someone takes the time to create them. Just reference it in your PR

**Comment by IndrajeetPatil** [comment]
Yeah, but I am not ignoring them by closing them. I closed the issues and immediately created comments in the respective Discussion.

Apologies to @PavelBal if he felt that I ignored his comments.

Every new issue that is created with respect to the pending PR is a new thing for me to keep track of. 

I am currently juggling:
- _ospsuite_ issues
- Discussion in _ospsuite_
- _tlf_ issues,
- PR comments
- personal correspondence
- Pavel's original proposal docs for these plots

So the narrower the surface area we keep for specifications, the more probable it is that I don't miss something.
